airbyte-integrations/connectors/destination-snowflake-cortex/integration_tests/spec.json

```diff
@@ -0,0 +1,411 @@
+              "type": "object",
+              "properties": {
+                "from_field": {
+                  "title": "From field name",
+                  "description": "The field name in the source",
+                  "type": "string"
+                },
+                "to_field": {
+                  "title": "To field name",
+                  "description": "The field name to use in the destination",
+                  "type": "string"
+                }
+              },
+              "required": ["from_field", "to_field"]
+            }
+          }
+        },
+        "required": ["chunk_size"],
+        "group": "processing"
+      },
+      "omit_raw_text": {
+        "title": "Do not store raw text",
+        "description": "Do not store the text that gets embedded along with the vector and the metadata in the destination. If set to true, only the vector and the metadata will be stored - in this case raw text for LLM use cases needs to be retrieved from another source.",
+        "default": false,
+        "group": "advanced",
+        "type": "boolean"
+      },
+      "indexing": {
+        "title": "Indexing",
+        "type": "object",
+        "properties": {
+          "host": {
+            "title": "Host",
+            "description": "Enter the account name you want to use to access the database. This is usually the identifier before .snowflakecomputing.com",
+            "airbyte_secret": true,
+            "examples": ["AIRBYTE_ACCOUNT"],
+            "type": "string"
+          },
+          "role": {
+            "title": "Role",
+            "description": "Enter the role that you want to use to access Snowflake",
+            "airbyte_secret": true,
+            "examples": ["AIRBYTE_ROLE", "ACCOUNTADMIN"],
+            "type": "string"
+          },
+          "warehouse": {
+            "title": "Warehouse",
+            "description": "Enter the name of the warehouse that you want to sync data into",
+            "airbyte_secret": true,
+            "examples": ["AIRBYTE_WAREHOUSE"],
+            "type": "string"
+          },
+          "database": {
+            "title": "Database",
+            "description": "Enter the name of the database that you want to sync data into",
+            "airbyte_secret": true,
+            "examples": ["AIRBYTE_DATABASE"],
+            "type": "string"
+          },
+          "default_schema": {
+            "title": "Default Schema",
+            "description": "Enter the name of the default schema",
+            "airbyte_secret": true,
+            "examples": ["AIRBYTE_SCHEMA"],
+            "type": "string"
+          },
+          "username": {
+            "title": "Username",
+            "description": "Enter the name of the user you want to use to access the database",
+            "airbyte_secret": true,
+            "examples": ["AIRBYTE_USER"],
+            "type": "string"
+          },
+          "credentials": {
```

Comment:
Added the password in credentials section. I wanted to add a dropdown with just the "password" option, but that was not possible since dropdown with one option is not allowed, Since we have password as a second level field, we should be able to keep the same when we implement multiple authorization options.

Reply:
This is great! And yes, makes sense! 👍